finat/fiat_elements.py

```diff
@@ -267,13 +265,7 @@ def mapping(self):
             return result
 
 
-@singledispatch
 def point_evaluation(fiat_element, order, refcoords, entity):
```

Comment:
You could just push this implementation into the body of the `point_evaluation` method now if you think it would be worth it.

Reply:
The best kind